src/nunit.analyzers/ClassicModelAssertUsage/ClassicModelAssertUsageCodeFix.cs

```diff
@@ -60,9 +60,13 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             if (arguments.Count > 1)
                 arguments[1] = CastIfNecessary(arguments[1]);
 
+            var argumentNamesToArguments = arguments.ToDictionary(
+                argument => GetParameterName(argument),
```

Comment:
Calling GetParameter for every argument which goes through all arguments each time is not efficiënt. It would be better to loop through the arguments and use the index to obtain the parameter name.